src/conn/conn_api.c

```diff
@@ -1283,6 +1283,12 @@ __conn_open_session(WT_CONNECTION *wt_conn, WT_EVENT_HANDLER *event_handler, con
     WT_ERR(__wt_open_session(conn, event_handler, config, true, &session_ret));
     *wt_sessionp = &session_ret->iface;
 
+#ifdef HAVE_CALL_LOG
+    if (F_ISSET(conn, WT_CONN_CALL_LOG_ENABLED)) {
```

Comment:
I am not sure we need to check this flag here.

If for some reason the call_log setup fails such that this flag is not set, then the connection is closed. Hence, the `open_session` function would inaccessible.

Reply:
Good point, I previously had that there when I was calling __wt_call_log_open_session for the internal sessions.